mono/metadata/w32process-win32.c

```diff
@@ -81,7 +81,7 @@ ves_icall_System_Diagnostics_Process_ShellExecuteEx_internal (MonoW32ProcessStar
 	gboolean ret;
 
 	shellex.cbSize = sizeof(SHELLEXECUTEINFO);
-	shellex.fMask = (gulong)(SEE_MASK_FLAG_DDEWAIT | SEE_MASK_NOCLOSEPROCESS | SEE_MASK_UNICODE);
+	shellex.fMask = (gulong)(SEE_MASK_NOASYNC | SEE_MASK_NOCLOSEPROCESS | SEE_MASK_UNICODE);
```

Comment:
Why the flag change?

Reply:
Commit description has the full info, but tldr it's the same value and the old one is deprecated

Reply:
This is reasonable and Win32 stinks.
The documentation says they have the same value, but don't use one, use the other.